docs/template-gcp-kubernetes.md

```diff
@@ -0,0 +1,83 @@
+The SAFE template has the ability to deploy to [Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) on Google Cloud Platform, GCP, with minimum effort. The focus of the template is to set up the basic building blocks like creating the container, publishing the container and also create a simple deployment on kubernetes that exposes a kubernetes service. You probably want to, and should, add things like a proper hostname and https support. That can be done in multiple ways and is more of a kubernetes specific task than a SAFE task.
+
+## Quickstart
+
+If you are familiar with GCP and already have `gcloud` and `kubectl` configured and an existing kubernetes cluster all you need to do to get started is running these commands:
+
+```
+git init
+git add .
+git commit -m "Initial commit"
+dotnet new SAFE --deploy gcp-kubernetes
+fake build -t Deploy -e SAFE_CLUSTER=<kubernetes cluster>
```

Comment:
Let's remove the additional SAFE_CLUSTER variable from this invocation, and let it default to `safe-cluster`. We can add a paragraph on SAFE_CLUSTER optional variable below